src/Resolver/InjectionInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
```

Comment:
`declare(strict_types=1);` is missing.

Compare with:

```php
<?php
/**
 * @see       https://github.com/{org}/{repo} for the canonical source repository
 * @copyright Copyright (c) {year} Zend Technologies USA Inc. (http://www.zend.com)
 * @license   https://github.com/{org}/{repo}/blob/master/LICENSE.md New BSD License
 */

declare(strict_types=1);

namespace {namespace};

```

https://github.com/zendframework/maintainers/blob/master/template/src/ConfigProvider.php#L8

Reply:
Yes, intentionally. I'm planning a separate PR to add strict_types to the whole codebase.

Reply:
Created TODO: #41